### ISSUE TITLE
It is recommended to support setting the login proxy IP for each account

### ISSUE THREAD

**Issue by hhun** [saifalisew1508/TelegramAdderTool#10]
It is recommended to support setting the login `proxy IP` for each account

For example: `socks` / `http`

**Comment by saifalisew1508** [comment]
I don't think this is a very effective solution. I have many telegram accounts logged in my device and none of them got banned till date because I just keep those accounts logged in. As soon as I start any spam activities from those accounts, my accounts start getting banned one by one. However, it is very rare that all accounts get banned at once. I think that free proxies increase the risk of account getting banned. For this, you will have to use fresh and new proxies which are available for paid. I don't think that I should increase the cost of my software users for this small problem. But still, I will think about your problem later and I will think about adding it in the next features update with an alternative/non-mandatory option.
